src/components/common/FileUploadPdf.vue

```diff
@@ -33,6 +36,13 @@ export default class FileUploadPdf extends Vue {
   @Prop({ default: null }) readonly pageSize!: PageSizes
   @Prop({ required: true }) readonly userId!: string
 
+  /** Set `required` to true for the three variables below once the feature flag is removed:
+   * `documentClass`, `documentType`, `businessIdentifier`
+  */
+  @Prop({ default: null }) readonly documentClass!: string
+  @Prop({ default: null }) readonly documentType!: string
+  @Prop({ default: null }) readonly businessIdentifier!: string
+
   // Network service functions
   @Prop({ required: true })
   readonly getPresignedUrl!: (fileName: string) => Promise<PresignedUrlIF>
```

Comment:
FYI, I'll be moving the network functions to Legal Services module. Shouldn't make a difference.